frontend/src/routes/Governance/discovered/DiscoveredPolicies.tsx

```diff
@@ -49,7 +49,7 @@ function nameCell(item: DiscoverdPolicyTableItem): ReactNode {
         policyName: item.name,
         apiGroup: item.policies[0].apigroup,
         apiVersion: item.policies[0].apiversion,
-        policyNamespace: item.policies[0].cluster,
+        policyNamespace: null,
```

Comment:
Should we remove this in` NavigationPath.discoveredByCluster` look like nowhere to use

Reply:
Also `discoveredByCluster` group by apiGroup + kind + name so we don't need this my apolgies